CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.16.3)
-project(Elastix)
+project(ITKElastix)
```

Comment:
Aren't just built-in ITK component supposed to have ITK prefix in their name like this?

Reply:
Ah, so that's the reason! Thanks!  :-) So what specifically can go wrong when the CMake project of ITKElastix is named ITKElastix?

Reply:
I think that primary benefit is reduction of confusion. I don't remember whether a prefix in a remote module would throw a wrench in the ITK's build system.

Reply:
Yes, the `ITK` prefix is a marker that the module is a remote vs builtin module. The prefix is also used for Doxygen grouping, possibly others.

Reply:
Honestly, I did encounter CMake errors from include(ITKModuleExternal) further on in the file:

https://github.com/InsightSoftwareConsortium/ITKElastix/blob/05cde7d3f147342702ffdb2dd6e7cbbad7e92123/CMakeLists.txt#L99 

Can you possibly explain what that is for? I mean, is it useful for ITKElastix to do `include(ITKModuleExternal)`?

Reply:
It is necessary for many things :-). It sets up the module configuration:

https://github.com/InsightSoftwareConsortium/ITK/blob/master/CMake/ITKModuleExternal.cmake

Reply:
Thanks Matt. https://github.com/InsightSoftwareConsortium/ITK/blob/8e34f1078e13b4ae39d8aaa600ff936a9a3050bb/CMake/ITKModuleExternal.cmake#L2-L3 says:

> This is the preferred way to build an ITK module outside of the ITK source tree.

So for my understanding, is ITKElastix itself an ITK module?

Reply:
It is, but it's name within ITK is `Elastix`, not `ITKElastix`. Similarly to how `ITKMontage` is called [`Montage`](https://github.com/InsightSoftwareConsortium/ITKMontage/blob/v0.7.3/itk-module.cmake#L14).

Reply:
When I renamed project(Elastix) to project(ITKElastix), I got the following errors:

```
 CMake Error at F:/X/Src/I/ITK53RC4/CMake/ITKModuleKWStyleTest.cmake:52 (file):
  file failed to open for writing (Permission denied):

    /ITKKWStyleFiles.txt
Call Stack (most recent call first):
  F:/X/Src/I/ITK53RC4/CMake/ITKModuleMacros.cmake:192 (itk_module_kwstyle_test)
  F:/X/Src/I/ITK53RC4/CMake/ITKModuleExternal.cmake:181 (itk_module_impl)
  CMakeLists.txt:99 (include)

```
These came indirectly from calling `include(ITKModuleExternal)`, at CMakeLists line 99. Any idea what it was trying to do? Was it trying to run KWStyle on ITKElastix?

Reply:
The project name is expected to be the same as the module name.

Reply:
Well, it also provides a interface for C++, soon WebAssembly, so not just Python :-)

If it needs to be changed, perhaps `ITKElastixModule`, or ...